rewrite-java-11/src/main/java/org/openrewrite/java/isolated/ReloadableJava11Parser.java

```diff
@@ -123,59 +121,19 @@ private ReloadableJava11Parser(
-                        overrideClasspath.add(part.toString());
-                    }
-                }
-                // make sure the rewrite-java-lombok dependency comes first
-                boolean found = false;
-                for (int i = 0; i < overrideClasspath.size(); i++) {
-                    if (overrideClasspath.get(i).contains("rewrite-java-lombok")) {
-                        overrideClasspath.add(0, overrideClasspath.remove(i));
-                        found = true;
-                    }
-                }
-                if (!found) {
-                    // try to find `rewrite-java-lombok` using class loader
-                    URL resource = getClass().getClassLoader().getResource("org/openrewrite/java/lombok/OpenRewriteConfigurationKeysLoader.class");
-                    if (resource != null && resource.getProtocol().equals("jar") && resource.getPath().startsWith("file:")) {
-                        String path = Paths.get(URI.create(resource.getPath().substring(0, resource.getPath().indexOf("!")))).toString();
-                        overrideClasspath.add(0, path);
-                    } else {
-                        break LOMBOK;
-                    }
-                }
-                System.setProperty("shadow.override.lombok", String.join(File.pathSeparator, overrideClasspath));
-
-                Class<?> shadowLoaderClass = Class.forName("lombok.launch.ShadowClassLoader", true, getClass().getClassLoader());
-                Constructor<?> shadowLoaderConstructor = shadowLoaderClass.getDeclaredConstructor(
-                        Class.forName("java.lang.ClassLoader"),
-                        Class.forName("java.lang.String"),
-                        Class.forName("java.lang.String"),
-                        Class.forName("java.util.List"),
-                        Class.forName("java.util.List"));
-                shadowLoaderConstructor.setAccessible(true);
-
-                ClassLoader lombokShadowLoader = (ClassLoader) shadowLoaderConstructor.newInstance(
-                        getClass().getClassLoader(),
-                        "lombok",
-                        null,
-                        emptyList(),
-                        singletonList("lombok.patcher.Symbols")
-                );
-                lombokProcessor = (Processor) lombokShadowLoader.loadClass("lombok.core.AnnotationProcessor").getDeclaredConstructor().newInstance();
-                Options.instance(context).put(Option.PROCESSOR, "lombok.launch.AnnotationProcessorHider$AnnotationProcessor");
+                lombokProcessor = Lombok.createLombokProcessor(getClass().getClassLoader());
             } catch (ReflectiveOperationException ignore) {
                 // Lombok was not found or could not be initialized
             } finally {
-                annotationProcessors = lombokProcessor != null ? singletonList(lombokProcessor) : emptyList();
+                if (lombokProcessor != null) {
```

Comment:
inverse this if statement (so `lombokProcessor == null`).

Reply:
Not quite as easy because the `annotationProcessors` field is `final`. But we could of course have it as a mutable list.